README.md

```diff
@@ -3,4 +3,14 @@
 This microservice will be used by our main APIs to handle searching.
 
 ## Setup
-1. Create a `.env` file in this project's root (copy `.env.default` into the `.env` and then add values after the `=` for each of the keys)
\ No newline at end of file
+
+1. Create a `.env` file in this project's root (copy `.env.default` into the `.env` and then    add values after the `=` for each of the keys)
+
+## Tests
+
+Run `npm test`
+**NOTE:** For debugging errors with SQL queries, please add this code `verbose: console.log` to the following object (`{memory: true}`) being passed in as a parameter for new sqliteDatabase() located in `searchDao.js`, AND all **Spec js files in the test directory.
```

Comment:
You did some backtick-quoting, but the bolding at the end still runs until the end of the paragraph instead of being explicitly closed.